langlib/langlib-test/src/test/resources/test-src/valuelib_fromJson_test.bal

```diff
@@ -828,6 +828,29 @@ function testFromJsonWithTypeToUnionOfTypeReference() {
     assertEquality(t3, <BoundAssertion> ["let", 3]);
 }
 
+function testFromJsonStringWithUnexpectedChars() {
+    string s1 = "{\"a\":}";
+    string s2 = "[1, 2,]";
+    string s3 = "{\"a\":1,}";
+    string s4 = "{\"a\": [1, 2,]}";
+
+    error err = <error> s1.fromJsonStringWithType(json);
+    assertEquality(<string> checkpanic err.detail()["message"], "expected 'a field value' at line: 1 column: 6");
```

Comment:
It is better if we don't have quotes around 'a field value' and 'an array element' (for characters it is okay). i.e. `"expected a field value at line: 1 column: 6"`, `"expected an array element at line: 1 column: 9"`. Shall we change the `StateMachine.throwExpected` method to support that?

Reply:
+1